projects/1.16.1/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2083 @@
+    "tetra/schematic/double/shared_head/hone_pickaxe_efficiency_5.name": "打磨：挖掘效率 V",
+    "tetra/schematic/double/shared_head/hone_pickaxe_efficiency_5.description": "进一步打磨组件的挖掘效率增益。",
+    "tetra.improvement.head_hone/pickaxe_efficiency.name": "打磨：挖掘效率",
+    "tetra.improvement.head_hone/pickaxe_efficiency.description": "增加挖掘效率",
+    "tetra/schematic/double/shared_head/hone_pickaxe_tier_1.name": "打磨：挖掘等级 I",
+    "tetra/schematic/double/shared_head/hone_pickaxe_tier_1.description": "打磨组件，使挖掘等级增加 1。",
+    "tetra.improvement.head_hone/pickaxe_tier.name": "打磨：挖掘等级",
+    "tetra.improvement.head_hone/pickaxe_tier.description": "使挖掘等级增加 1",
+    "tetra/schematic/double/shared_head/hone_hammer_tier_1.name": "打磨：锤子等级 I",
+    "tetra/schematic/double/shared_head/hone_hammer_tier_1.description": "打磨组件，使锤子等级增加 1。",
+    "tetra.improvement.head_hone/hammer_tier.name": "打磨：锤子等级",
+    "tetra.improvement.head_hone/hammer_tier.description": "增加锤子等级 1",
+    "tetra.synergy.double/axe_hammer": "木工锤",
+    "tetra.synergy.double/axe_axe": "巨斧",
+    "tetra.synergy.double/axe_pickaxe": "錾斧",
+    "tetra.synergy.double/hammer_pickaxe": "地质锤",
+    "tetra.synergy.double/adze_pickaxe": "鹤嘴锄",
+    "tetra.synergy.double/adze_axe": "切割型鹤嘴锄",
+    "tetra.synergy.double/claw_hammer": "羊角锤",
+    "tetra.synergy.double/claw_axe": "木匠斧",
+    "tetra/schematic/double/basic_handle/basic_handle.name": "基础手柄",
+    "tetra/schematic/double/basic_handle/basic_handle.description": "替换工具的手柄。坚固的手柄能够支持更重、更高级的工具头部。\n \n材料§f密度§r略微减少§f攻击速度§r。",
+    "tetra/schematic/double/basic_handle/basic_handle.slot1": "手柄材料",
+    "tetra.module.double/basic_handle.name": "基础手柄",
+    "tetra.module.double/basic_handle.description": "基础工具手柄。手柄是工具完整度的主要来源。",
+    "tetra.module.double/basic_handle.material_name": "%s手柄",
+    "tetra.variant.basic_handle/forged_beam": "再利用梁杆",
+    "tetra.improvement.basic_handle/blazing.name": "燃烧",
+    "tetra.improvement.basic_handle/blazing.description": "你可能在使用时着火，尤其是在炎热的环境中。",
+    "tetra.improvement.basic_handle/ender_reverb.name": "末影回响",
+    "tetra.improvement.basic_handle/ender_reverb.description": "在附近有末地生物时产生副作用。",
+    "tetra.improvement.basic_handle/wrap_string.name": "绳裹柄",
+    "tetra.improvement.basic_handle/wrap_string.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.basic_handle/wrap_leather.name": "皮革裹柄",
+    "tetra.improvement.basic_handle/wrap_leather.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.basic_handle/wrap_hide.name": "兔皮裹柄",
+    "tetra.improvement.basic_handle/wrap_hide.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra.improvement.basic_handle/wrap_vine.name": "藤蔓裹柄",
+    "tetra.improvement.basic_handle/wrap_vine.description": "轻微增加攻击和攻速，减少手柄负面效果。",
+    "tetra/schematic/double/basic_handle/wrap_handle.name": "裹柄",
+    "tetra/schematic/double/basic_handle/wrap_handle.slot1": "材料",
+    "tetra/schematic/double/basic_handle/wrap_handle.description": "使用给定材料裹柄。轻微增加攻击和攻速，并减少手柄可能导致的负面效果。",
```

|Feedback Priority|
|---|
|⚪️ Pebble / Non-blocking but requires future action|

```suggestion
    "tetra/schematic/double/basic_handle/wrap_handle.description": "使用给定材料裹柄。较小地增加伤害和挥动速度，并且减少手柄可能导致的负面效果。",
```
同 L582